_data/users.yml

```diff
@@ -194,6 +194,9 @@
 - name: MoEngage
   logo: /assets/images/logos/moengage.png
   logosmall: /assets/images/logos/moengage-small.png
+- name: Myantra
```

Comment:
Supposed to be `Myntra` IIRC - https://www.myntra.com/

Reply:
Made an issue, I'll fix